src/main/java/com/project/backend/ClassDay.java

```diff
@@ -0,0 +1,55 @@
+package com.project.backend;
+
+import java.util.ArrayList;
+import java.util.Date;
```

Comment:
The `Date` implementation in Java is a bit... broken. If possible we should probably find an alternative for this - probably the `java.time` package and its classes/methods.

See http://stackoverflow.com/questions/1969442/whats-wrong-with-java-date-time-api for some rationale.